optuna/importance/_fanova/_evaluator.py

```diff
@@ -126,6 +126,8 @@ def evaluate(
         param_importances = numpy.array(
             [evaluator.get_importance(i)[0] for i in range(len(non_single_distributions))]
         )
+        # NOTE(nabenabe0928): Normalize is here to keep the backward compatibility.
+        # TODO(nabenabe0928): Add the `normalize` argument optionally.
```

Comment:
I agree that there doesn't seem to be a particular consensus on this, shall we delete this comment?

Reply:
```suggestion
```